packages/react/src/autocomplete-item/AutocompletItem.tsx

```diff
@@ -8,47 +8,48 @@ import { extractSprinkles } from "../sprinkles";
 import { fallbackSpan } from "../utils";
 import * as styles from "./AutocompleteItem.css";
 
-type AccordionItemProps = BoxProps<
+type AutocompleteItemProps = BoxProps<
   "li",
   {
     addonBefore?: ReactNode;
     item: unknown;
   }
 >;
 
-export const AutocompleteItem = forwardRef<HTMLLIElement, AccordionItemProps>(
-  ({ addonBefore, children, item, ...props }, ref) => {
-    const { restProps, sprinkleProps } = extractSprinkles(props);
-    const { downshift, highlightedItem } =
-      useAutocompleteContext("AutocompleteItem");
+export const AutocompleteItem = forwardRef<
+  HTMLLIElement,
+  AutocompleteItemProps
+>(({ addonBefore, children, item, ...props }, ref) => {
+  const { restProps, sprinkleProps } = extractSprinkles(props);
+  const { downshift, highlightedItem } =
+    useAutocompleteContext("AutocompleteItem");
 
-    return (
-      <Flex asChild {...styles.item()} {...sprinkleProps}>
-        <li
-          data-disabled={
-            downshift.getItemProps({ item })["aria-disabled"] ? "" : undefined
-          }
-          data-highlighted={highlightedItem === item ? "" : undefined}
-          data-selected={downshift.selectedItem === item ? "" : undefined}
-          ref={ref}
-          {...restProps}
-          {...downshift.getItemProps({
-            item,
-          })}
-        >
-          {addonBefore && (
-            <Box asChild h="16" w="auto">
-              {fallbackSpan(addonBefore)}
-            </Box>
-          )}
+  return (
+    <Flex asChild {...styles.item()} {...sprinkleProps}>
+      <li
+        data-disabled={
+          downshift.getItemProps({ item })["aria-disabled"] ? "" : undefined
+        }
+        data-highlighted={highlightedItem === item ? "" : undefined}
+        data-selected={downshift.selectedItem === item ? "" : undefined}
+        ref={ref}
+        {...restProps}
+        {...downshift.getItemProps({
+          item,
+        })}
+      >
+        {addonBefore && (
+          <Box asChild h="16" w="auto">
+            {fallbackSpan(addonBefore)}
+          </Box>
+        )}
 
-          {children}
+        {children}
 
-          {downshift.selectedItem === item && <IconCheck />}
-        </li>
-      </Flex>
-    );
-  },
-);
+        {downshift.selectedItem === item && <IconCheck />}
```

Comment:
can we make this optional? maybe move this to an `ItemIndicator` component

so `AutocompleteItemIndicator` - and consumers can use them if they want or opt-out

and all it does is render an icon if the parent item is the currently selected item

so we'd need an additional context here to pass like `active` or something

```
<AutocompleteItemContextProvider active={downshift.selectedItem === item}>
  <li>{...}</li>
</AutocompleteItemContextProvider>
```

and in `AutocompleteItemIndicator` we'd render null or icon depending on this context value